### PR TITLE
MAR-1800. Added tests to ButtonSlice

### PR DESCRIPTION
**Pull request by TangoPJ**
Link to the task: https://maddevs.atlassian.net/browse/MAR-1800

Added following tests to the ButtonSlice:
1. Test to check correctly render UIButton, UIOutlinedButton, UILinkButton
2. To computed methods:
      - background color
      - alignment
3. To data-aos animation attribute
4. Improved computed method to check alignment in the component
5. Updated snapshots

### CODE REVIEW

File: client/prismicSlices/pageParts/ButtonSlice/index.vue [hunk 1]
```diff
@@ -5,20 +5,22 @@
   >
     <div
       class="container"
-      :class="sliceAlignmentClass"
+      :class="`button-slice__content--align-${sliceAlignment}`"
+      :data-aos="animation"
+      data-testid="container"
```
Comment (kotyunin) [inline client/prismicSlices/pageParts/ButtonSlice/index.vue:10]: лучше button-slice-container

File: client/prismicSlices/pageParts/ButtonSlice/index.vue [hunk 2]
```diff
@@ -92,28 +94,28 @@ export default {
...
+      animation: this.slice?.primary?.animation,
     }
   },
 
   computed: {
     sliceBackground() {
-      if (this.slice.primary.background === 'white') return '#fff'
-      if (this.slice.primary.background === 'grey') return '#f5f7f9'
-      if (this.slice.primary.background === 'black') return '#111213'
+      if (this.slice?.primary?.background === 'white') return '#fff'
```
Comment (kotyunin) [inline client/prismicSlices/pageParts/ButtonSlice/index.vue:109]: #ffffff

File: tests/prismicSlices/pageParts/ButtonSlice/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
+import { render, screen } from '@testing-library/vue'
+import { shallowMount } from '@vue/test-utils'
+import ButtonSlice from '@/prismicSlices/pageParts/ButtonSlice'
+
+const backgrounds = {
+  white: '#fff',
```
Comment (kotyunin) [inline tests/prismicSlices/pageParts/ButtonSlice/index.spec.js:6]: '#ffffff'

File: client/prismicSlices/pageParts/ButtonSlice/index.vue (hunk 1 above)
Comment (TangoPJ) [reply client/prismicSlices/pageParts/ButtonSlice/index.vue:10]: done

File: client/prismicSlices/pageParts/ButtonSlice/index.vue (hunk 2 above)
Comment (TangoPJ) [reply client/prismicSlices/pageParts/ButtonSlice/index.vue:109]: done

File: tests/prismicSlices/pageParts/ButtonSlice/index.spec.js (hunk 3 above)
Comment (TangoPJ) [reply tests/prismicSlices/pageParts/ButtonSlice/index.spec.js:6]: done

**Pull request merged**